tasks/coverage/snapshots/parser_typescript.snap

```diff
@@ -4730,23 +4730,32 @@ Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/salsa/private
    ·          ────────────────
    ╰────
 
+  × TS(1248): A class member cannot have the 'const' keyword.
+   ╭─[typescript/tests/cases/compiler/ClassDeclaration26.ts:2:18]
+ 1 │ class C {
+ 2 │     public const var export foo = 10;
+   ·                  ───
```

Comment:
This is pointing at the wrong span.

Reply:
I agree, I did not add checks for duplicate declaration keywords. I'm thinking this should go in another PR. Thoughts?

Reply:
Sure.